Dshop/apps/users/tests/test_api_login.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+User = get_user_model()
+
+
+@pytest.fixture
+def api_client():
+    return APIClient()
+
+
+@pytest.fixture
+def login_url():
+    return reverse('api-login')
+
+
+@pytest.fixture
+def login_data():
+    return {
+        'username': 'testuser',
+        'password': 'testpassword',
+    }
+
+
+@pytest.fixture
+def user_instance(login_data):
+    return User.objects.create_user(**login_data)
+
+
+@pytest.mark.django_db
+def test_login_success(api_client, login_url, login_data, user_instance):

```

Comment:
Good test cases.
I am thinking, I would also check what is the returned data in case of success.